spec/schemas/MerkleStateInfoDTO.yml

```diff
@@ -3,6 +3,14 @@ description: |
   The merkle path information clients can use to proof the state of the given entity.
 required:
   - raw
+  - tree
 properties:
   raw:
     $ref: "./MerkleTreeRaw.yml"
+  tree:
```

Comment:
I would split tree into branches (list of MerkleTreeBranchDTO) and leaf (optional MerkleTreeLeafDTO) removing the anyOf and simplifying the mappers into 1-1 in the SDKs

Reply:
Only if it makes sense, we only have 1 leaf at most.

Reply:
After last conversation, we have decided to go with a generic list of branches/leaves. Only one leaf is expected in a valid case, but it's good to have a generic structure